### ISSUE TITLE
ORTCDemo: focus state for "connect" button is not obvious enough

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#196]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/ortcdemo/

Changes from transparent background to white, which is very similar. 

It looks like the styles are busted as no top margin (owl removal probably) and the hover state is blue.

**Comment by alrra** [comment]
@dstorey @sarvaje This demo is external, namely, it's from https://ortcdemo.azurewebsites.net.

**Comment by molant** [comment]
We have access to that server. Don't know where the code is but we can modify the templates directly if needed. Erik should know more and @sarvaje should have access to it via the azure portal.

**Comment by sarvaje** [comment]
Yep, I have access to the server so I can change the css directly. @dstorey is it good just to change the current focus style for the hover style (the blue color)

**Comment by dstorey** [comment]
@sarvaje it should probably have something similar to the edgeCore focus styles, as if use same as hover it would be obvious that they're different. Maybe @melanierichards can chime in.
